C4SGWeb/app/project/view/project-view.component.ts

```diff
@@ -11,27 +12,50 @@ import { ProjectService } from '../common/project.service';
 })
 
 export class ProjectViewComponent implements OnInit {
-
+  projectForm = FormGroup;
   project: Project;
   params: Params;
+  locations = [
+    'Remote',
+    'Physical Address'
+    ];
+  data = {
+    name: 'name',
+    org: 'org',
+    date: 'date',
+    description: 'description',
+    location: 'location',
+    address: 'address',
+    state: 'state',
+    city: 'city',
+    zip: 'zip',
+    country: 'country',
+  };
 
   constructor(private projectService: ProjectService,
               private route: ActivatedRoute,
               private router: Router,
-              private location: Location) {
+              private location: Location,
+              @Inject(FormBuilder) fb: FormBuilder) {
```

Comment:
Oh yeah that doesn't need to be there. It can be removed. Is there a way for me to delete that without submitting another pull request?